start_cerberus.py

```diff
@@ -78,6 +84,10 @@ def main(cfg):
             logging.info("Detailed inspection of failed components has been enabled")
             inspect.delete_inspect_directory()
 
+        # Variables used for multiprocessing
+        pool = multiprocessing.Pool(multiprocessing.cpu_count())
```

Comment:
Let's add a way to specify the percentage of cores to be used for the multiprocessing to avoid hogging up all the resources on the machine causing other jobs to slow down. Maybe use 50% of the available CPU cores by default and an option to bump it up if user is okay with it? Thoughts?

Reply:
I think we can use 75% by default.